driver/pgxv5/postgres_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgxv5_test
+
+import (
+	"database/sql"
+	"log"
+	"time"
+
+	"cloud.google.com/go/alloydbconn/driver/pgxv5"
+)
+
+// Example shows how to use the AlloyDB driver
+func ExampleRegisterDriver() {
```

Comment:
This can go -- and so can the v4 example. It's not used.

Reply:
Removed.